content/aro/acm-odf-aro/index.md

```diff
@@ -0,0 +1,1135 @@
+
+```az group create                		\
+  --name $AZR_RESOURCE_GROUP   	        \
+  --location $AZR_RESOURCE_LOCATION
+  ```
+
+  3. Create an Azure Service Principal
+
+  ```
+  AZ_SUB_ID=$(az account show --query id -o tsv)
+AZ_SP_PASS=$(az ad sp create-for-rbac -n "${AZR_CLUSTER}-SP" --role contributor \
+  --scopes "/subscriptions/${AZ_SUB_ID}/resourceGroups/${AZR_RESOURCE_GROUP}" 	\
+  --query "password" -o tsv)
+AZ_SP_ID=$(az ad sp list --display-name "${AZR_CLUSTER}-SP" --query "[0].appId" -o tsv)
+```
+
+
+### Networking
+
+Create a virtual network with two empty subnets
+
+1. Create virtual network
+
+```
+az network vnet create                                    	\
+  --address-prefixes $VIRTUAL_NETWORK                    	\
+  --name "$AZR_CLUSTER-aro-vnet-$AZR_RESOURCE_LOCATION"   	\
+  --resource-group $AZR_RESOURCE_GROUP
+```
+
+2. Create control plane subnet
+
+```  az network vnet subnet create                                  \
+  --resource-group $AZR_RESOURCE_GROUP                            	\
+  --vnet-name "$AZR_CLUSTER-aro-vnet-$AZR_RESOURCE_LOCATION"      	\
+  --name "$AZR_CLUSTER-aro-control-subnet-$AZR_RESOURCE_LOCATION" 	\
+  --address-prefixes $CONTROL_SUBNET 
+  ```
+
+3. Create worker subnet
+
+```  az network vnet subnet create                                \
+  --resource-group $AZR_RESOURCE_GROUP                            \
+  --vnet-name "$AZR_CLUSTER-aro-vnet-$AZR_RESOURCE_LOCATION"      \
+  --name "$AZR_CLUSTER-aro-worker-subnet-$AZR_RESOURCE_LOCATION"  \
+  --address-prefixes $WORKER_SUBNET   
+  ```
+
+### Jump Host
+
+With the cluster in a private network, we can create a jump host in order to connect to it. 
+
+1. Create the jump subnet
+
+```az network vnet subnet create                                \
+  --resource-group $AZR_RESOURCE_GROUP                       	\
+  --vnet-name "$AZR_CLUSTER-aro-vnet-$AZR_RESOURCE_LOCATION" 	\
+  --name JumpSubnet                                          	\
+  --address-prefixes $JUMPHOST_SUBNET    
+  ```
+
+2. Create a jump host
+
+``` az vm create --name jumphost                 		\
+    --resource-group $AZR_RESOURCE_GROUP     	        \
+    --ssh-key-values $HOME/.ssh/id_rsa.pub   	        \
+    --admin-username aro                     			\
+    --image "RedHat:RHEL:9_1:9.1.2022112113" 		    \
+    --subnet JumpSubnet                      			\
+    --public-ip-address jumphost-ip          		    \
+    --public-ip-sku Standard                 			\
+    --vnet-name "$AZR_CLUSTER-aro-vnet-$AZR_RESOURCE_LOCATION"
+```
+
+3. Save the jump host public IP address
+
+```JUMP_IP=$(az vm list-ip-addresses -g $AZR_RESOURCE_GROUP -n jumphost -o tsv \
+--query '[].virtualMachine.network.publicIpAddresses[0].ipAddress')
+
+echo $JUMP_IP
+```
+
+4. Use sshuttle to create a SSH VPN via the jump host (use a separate terminal session)
+
+Replace the IP with the IP of the jump box from the previous step
+
+```
+sshuttle --dns -NHr "aro@${JUMP_IP}"  10.0.0.0/8
```

Comment:
Change 10.0.0.0/8 to an environment variable .. for example I have a conflict with 10.0.x so am using 10.1.x

Reply:
Running the above command with sshuttle doesn't allow you to run the create cluster command.  Add steps to run the sshuttle in the background.   Also, do we really need to run sshuttle at this point?